pkg/addon/policyframework/agent_addon.go

```diff
@@ -83,6 +89,10 @@ func getValues(cluster *clusterv1.ManagedCluster,
 				PkgLogLevel: -1,
 			},
 			SyncPoliciesOnMulticlusterHub: false,
+			// Defaults from `values.yaml` will be used if these stay at 0.
```

Comment:
I know I did exactly this (without a test) in the config-policy section, but can we confirm this via the expected args in the `should create the default framework deployment` test?